src/Systmm.f90

```diff
@@ -397,9 +401,9 @@ SUBROUTINE SYSTMM(temp_file)
       4650 format(16x,12(6x,f6.0,6x))
       4700 format(f10.4,f6.0,15(f6.1,f8.3))
       4750 format(f10.4,10(i4,f8.0))
-      
+[THESE FORMAT STATEMENTS ARE PROBABLY VESTIGES OF AN EARLIER VERSION AND UNNECESSARY - JRY]      
       temp_out(:) = T_res(:) !set reservoir temperature for next time step
-
+[I'M  NOT SURE YOUR CAN DO THIS WITHOUT STORING ALL VALUES FOR BOTH TIME = N and TIME = N+1 - JRY] 
```

Comment:
The idea with this code is "temp_out" is the temperature of the water leaving the reservoir - so in the next day loop if the particle is tracked to a reservoir, it can use "temp_out". I realize now that this should be down below with the "write(32...", so it's only recorded after all reservoirs have been simulated.